lms/static/scripts/frontend_apps/utils/test/api-test.js

```diff
@@ -30,6 +30,29 @@ describe('api', () => {
       });
     });
 
+    it('sets query params if `params` is passed', async () => {
```

Comment:
Do the existing tests exercise the path of not having any `params` and proving that the resulting URL is OK?

Reply:
Yes. The other tests don't set `params`.